### ISSUE TITLE
Add support for g:vimwiki_listsyms

### ISSUE THREAD

**Issue by canaaerus** [tbabej/taskwiki#119]
In vimwiki the symbols inside the checkboxes of the tasks can be customized by setting `g:vimwiki_listsyms` to a string of 5 symbols corresponding to different completion states of the task from 0% to 100%. E.g. I have set

    let g:vimwiki_listsyms = ' ▂▄▆✓'

Unfortunately taskwiki does not recognize these.

Furthermore using the vimwiki key combination `<C-Space>` will set the customized symbol (`✓`) and prevents taskwiki from recognizing the current and any following tasks as belonging to a viewport.

**Comment by tbabej** [comment]
What do you mean by not recognizing them? Tasks with these symbols in the brackets are not added/updated to taskwarrior?

As far as the done checkmark symbol goes, for that we will need to use the variable from vim to  determine what symbol means 'done' task, instead of currently hardcoded 'X'.

**Comment by canaaerus** [comment]
Exactly, curiously, any task which is marked in the above fashion as done or partially done is not added/updated to taskwarrior. From looking at the regex `COMPLETION_MARK` I would have expected otherwise.

It seems that using the default symbols `' .oOX'` works ok. So the problem is most likely related to me choosing fancy unicode symbols with more than 1 byte in their UTF-8 representation.

Btw. I am using python 3.4.5.

**Comment by tbabej** [comment]
Yes, that's my suspicion as well. Would you mind trying modifying the source of regex definition like this?

    -GENERIC_TASK = re.compile(''.join([
    +GENERIC_TASK = re.compile(u''.join([
         '^',
         ...
         '$'    # Enforce match on the whole line
    -]))
    +]), re.UNICODE)

We would also need tests for this. By the way, writing tests for taskwiki is pleasure.

**Comment by canaaerus** [comment]
That did not work, but I debugged some more. I actually have python2 as well as python3 support enabled, so the `:py` command is using python2.7. Now the lines in the taskwiki buffer are actually utf-8 encoded byte-strings. Thus `✓` is two bytes long and is not matched by a single `.`.

**Comment by tbabej** [comment]
Well, this is where this gets tricky. From what I played around, it seems that what you actually get out of buffer is dependent on whether you use neovim/vim and whether you use py3/py2. So far we survived this predicament since we 'put back in what we got'. But it would be nice to simplify stuff by building a layer on top of that, and deal only with, say, utf-8 strings in the buffer from taskwiki's point of view.

**Comment by tbabej** [comment]
Python 2 support will be dropped in the next releases, so this might be easier to resolve.
